specificatie/genereervariant/openapi.json

```diff
@@ -314,6 +314,72 @@
               "type": "string"
             }
           },
+          {
+            "name": "postcode",
+            "in": "query",
+            "description": "Postcode behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "pattern": "^[1-9]{1}[0-9]{3}[a-zA-Z]{2}$",
+              "type": "string",
+              "example": "6922KZ"
+            }
+          },
+          {
+            "name": "huisnummer",
+            "in": "query",
+            "description": "Huisnummer behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "maximum": 99999,
+              "minimum": 1,
+              "type": "integer",
+              "example": 1
+            }
+          },
+          {
+            "name": "huisletter",
+            "in": "query",
+            "description": "Huisletter behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "pattern": "^[a-zA-Z]{1}$",
+              "type": "string",
+              "example": "A"
+            }
+          },
+          {
+            "name": "huisnummertoevoeging",
+            "in": "query",
+            "description": "Huisnummertoevoeging behorende bij een nummeraanduiding.",
+            "required": false,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "pattern": "^[0-9a-zA-Z]{1,4}$",
+              "type": "string",
+              "example": "bis"
+            }
+          },
+          {
+            "name": "exacteMatch",
+            "in": "query",
+            "description": "\"Een indicatie of de resultaten van een zoekoperatie exact overeen moeten komen met de zoekcriteria, zie [functionele specificatie exacte match](https://github.com/lvbag/BAG-API/blob/master/Features/exacte_match.feature).\"",
```

Comment:
Geeft aan of de resultaten van de zoekopdracht exact overeen moeten komen met de opgegeven parameters, zie ...

Reply:
klopt deze link: https://github.com/lvbag/BAG-API/blob/master/Features/exacte_match.feature?

ik zie de map "features" (kleine F)